google-cloud-bom/pom.xml

```diff
@@ -620,6 +620,16 @@
         <artifactId>grpc-google-cloud-securitycenter-v1beta1</artifactId>
         <version>0.47.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-securitycenter-v1beta1:current} -->
       </dependency>
+      <dependency>
```

Comment:
Please also add a reference to `google-cloud-clients/google-cloud-securitycenter`

Reply:
Fixed